### PR TITLE
BLOCKCHAIN-65 - Identity Office FE - distribute LLM/LLD after confirming citizenship

### PR DESCRIPTION
**Pull request by MichalLustykNeti**

### CODE REVIEW

**Review by kacperzuk-neti — APPROVED** [review]

File: src/components/Offices/Identity/index.js [hunk 1]
```diff
@@ -187,41 +186,44 @@ function IdentityTable({ info }) {
...
-  identity = identity.unwrap();
-  const { hash } = identity.info;
+  if (onchain === null) return null;
+  if (onchain.isNone) return <MissingIdentity />;
+
+  onchain = onchain.unwrap();
+  const { hash } = onchain.info;
+
+  const backendMerits = backend?.merits ?? 0
+  const backendDollars = backendMerits?.mul(10);
```
Comment (kacperzuk-neti) [inline src/components/Offices/Identity/index.js:204]:
```suggestion
  const backendDollars = backendMerits?.div(10);
```
It's 10 LLM -> 1 LLD.

**Review by kacperzuk-neti — CHANGES_REQUESTED** [review]

**Review by kacperzuk-neti — APPROVED** [review]

File: src/components/Offices/Identity/index.js [hunk 2]
```diff
@@ -21,7 +21,6 @@ function IdentityForm() {
 
   const onSubmit = ({ account }) => {
     dispatch(officesActions.officeGetIdentity.call(account));
-    dispatch(officesActions.getBackendAddressLlm.call({ walletAddress: account }));
```
Comment (kacperzuk-neti) [inline src/components/Offices/Identity/index.js:24]: This data is now fetched via `officesActions.officeGetIdentity`.

File: src/components/Offices/Identity/index.js [hunk 3]
```diff
@@ -187,41 +186,44 @@ function IdentityTable({ info }) {
...
-  identity = identity.unwrap();
-  const { hash } = identity.info;
+  if (onchain === null) return null;
+  if (onchain.isNone) return <MissingIdentity />;
+
+  onchain = onchain.unwrap();
+  const { hash } = onchain.info;
+
+  const backendMerits = backend?.merits ?? 0
+  const backendDollars = backendMerits?.div(10);
```
Comment (kacperzuk-neti) [inline src/components/Offices/Identity/index.js:204]: This will round down to full LLD, is that OK?

File: src/api/backend.js [hunk 4]
```diff
@@ -12,14 +13,29 @@ const getApi = () => {
...
+  }))
 };
+
+export const addMeritTransaction = async (userId, amount) => {
+  const api = getApi();
+
+  const formattedAmount = ethers.utils.formatUnits(amount, 12);
+  await api.post("/merit-transactions", {
+    userId,
+    amount: formattedAmount,
```
Comment (DorianSternVukotic) [inline src/api/backend.js:37]: this might need to be minus amount ?

Comment (kacperzuk) [reply src/api/backend.js:37]: Minus is added here: https://github.com/liberland/liberland_frontend/pull/99/files#diff-372c7d46a63791bfa3dbddf18062c2e107720f213d4e260f16eef16f0ac21044R52

**Pull request merged**